docs/design/versioning.md

```diff
@@ -68,6 +68,12 @@ Here is an example major release cycle:
 
 It may seem a bit strange to complete the v2 API before v2.0 is released, but *adding* a v2 API is not a breaking change. *Removing* the v2beta\* APIs *is* a breaking change, which is what necessitates the major version bump. There are other ways to do this, but having the major release be the fresh start of that release's API without the baggage of its beta versions seems most intuitive out of the available options.
 
+# Patches
+
+GKE only allows users to run at the latest path (x.x.**X**) of any given minor release (x.**X**.x). From a customer perspective, this means there should be no reason a customer would ever want to run a non-latest patch, as it will be impossible (via GKE).
```

Comment:
Can we make this not GKE specific?

cc @bgrant0607 

(Nit: I assume you meant `latest patch`?)